packages/docusaurus/lib/client/exports/ComponentCreator.js

```diff
@@ -63,8 +62,8 @@ function ComponentCreator(path) {
     modules: optsModules,
     webpack: () => optsWebpack,
     render: (loaded, props) => {
-      // Transform back loaded modules back into the original structure.
-      const loadedModules = originalModules;
+      // clone the original object since we don't want to alter the original.
+      const loadedModules = JSON.parse(JSON.stringify(modules));
```

Comment:
Interesting. I thought I was using `_.cloneDeep` here. If we import just the function it's fine:

```
import cloneDeep from 'lodash/cloneDeep`; // It's ok, just cloneDeep is imported
// vs
import _ from 'lodash`; // Not ok, the entire lodash is imported
```

Reply:
Sry I removed it before because bundle size is still very big. Few KBs saved

Reply:
#1367